projects/core/components/link/link.component.ts

```diff
@@ -67,8 +67,10 @@ export class TuiLinkComponent implements TuiFocusableElementAccessor {
     focusVisible = false;
 
     readonly focusedChange = merge(
-        tuiTypedFromEvent(this.elementRef.nativeElement, 'focusin').pipe(mapTo(true)),
-        tuiTypedFromEvent(this.elementRef.nativeElement, 'focusout').pipe(mapTo(false)),
+        tuiTypedFromEvent(this.elementRef.nativeElement, 'focusin').pipe(map(() => true)),
```

Comment:
```suggestion
        tuiTypedFromEvent(this.elementRef.nativeElement, 'focusin').pipe(map(ALWAYS_TRUE_HANDLER),
```

https://github.com/Tinkoff/taiga-ui/blob/4b799b5f5fb98eeda678b5aa7af82f9901544667/projects/cdk/constants/always-true-handler.ts#L4

Reply:
fixed